### PR TITLE
Update on DG and AboutUs

### LINKED ISSUE #20: Update AboutUs page
1. Add your own details
2. Include a suitable photo
3. Indicate the different roles played and responsibilities held by each team member

- [x] @Pluiexo  
- [x] @tsulim 
- [x] @whitesnowx 
- [x] @JerryWang0000 
- [x] @iynixil

### LINKED ISSUE #24: Add Use-Cases to the DeveloperGuide :file_folder:
Use Case for 
- [x] Add
- [x] Filter
- [x] Sort
- [x] Delete
- [x] Edit

### PR DESCRIPTION
**Pull request by whitesnowx**
Update Use Cases in DG
Update name to lowercases in aboutus

### CODE REVIEW

File: docs/team/calistawong.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: John Doe's Project Portfolio Page
+title: Wong's Project Portfolio Page
```
Comment (tsulim) [inline docs/team/calistawong.md:3]: Just checking, did you miss out your name in this portion?

**Comment by whitesnowx** [comment]
HELLO ALL, DONT APPROVE THIS YET

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -293,30 +293,98 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+1. StaffConnect shows a list of persons
+2. User requests to sort the list by a specific attribute
+3. StaffConnect shows a sorted list of persons
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
```
Comment (tsulim) [inline docs/DeveloperGuide.md]: I think you made a typo here. it should be `1a` instead of `2a`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -293,30 +293,98 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. StaffConnect shows a list of persons
+2. User requests to sort the list by a specific attribute
+3. StaffConnect shows a sorted list of persons
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
```
Comment (tsulim) [inline docs/DeveloperGuide.md:329]: Same typo found here. Please correct me if I made a mistake.

**Review by tsulim — COMMENTED** [review]
LGTM! Just some typos

File: docs/team/calistawong.md (hunk 1 above)
Comment (whitesnowx) [reply docs/team/calistawong.md:3]: donnid to edit .md yet

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (whitesnowx) [reply docs/DeveloperGuide.md]: i just notice that whole use case is wrong, updating now

File: docs/team/calistawong.md (hunk 1 above)
Comment (tsulim) [reply docs/team/calistawong.md:3]: Oh thanks, got it 👍

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (whitesnowx) [reply docs/DeveloperGuide.md:329]: updated.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -310,18 +310,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-**Use case: Sort the list**
+**Use case: Filter the list**
 
 **MSS**
 
 1. StaffConnect shows a list of persons
-2. User requests to sort the list by a specific attribute
+2. User requests to filter the list by a specific attribute
 3. StaffConnect shows a sorted list of persons
```
Comment (tsulim) [inline docs/DeveloperGuide.md]: Perhaps this should be `filtered` instead of `sorted`?

**Review by tsulim — COMMENTED** [review]
LGTM. Just a few nits to fix.

**Review by tsulim — APPROVED** [review]
LGTM!

**Pull request merged**